server/pairing/client.go

```diff
@@ -131,14 +188,50 @@ func (c *Client) sendSyncDeviceData() error {
 	}
 
 	if resp.StatusCode != http.StatusOK {
+		err = fmt.Errorf("[client] status not okay when sending sync device data, status: %s", resp.Status)
 		signal.SendLocalPairingEvent(Event{Type: EventTransferError, Error: err.Error(), Action: ActionSyncDevice})
-		return fmt.Errorf("status not ok, received '%s'", resp.Status)
+		return err
 	}
 
 	signal.SendLocalPairingEvent(Event{Type: EventTransferSuccess, Action: ActionSyncDevice})
 	return nil
 }
 
+func (c *Client) receiveInstallationData() error {
```

Comment:
These functions seem to only send one side of the installation data. Forgive me if I've misunderstood your work, but it seems that only `receiving` device gets the installation data of its counterpart.

Where as the 2 devices should exchange their installation data.

For example a `receiving client` would call `receiveInstallationData()` then `POST` it's installation data and receive the `sending server`'s installation data in the http response.

In the case of a `sending client` it would call `sendInstallationData()` then `POST` it's installation data and receive the `sending server`'s installation data in the http response.

Ah, so actually, you only need one client and one server function for this behaviour, as both can be exchanged in a single HTTPS request/response . And the process flow for both should be identical.

@qfrank Does that sound right to you?

Reply:
My point was: `protobuf.PairInstallation` is also a kind of RawMessage so we can handle it within [function HandleSyncRawMessages](https://github.com/status-im/status-go/blob/6cbeedd880fd76a9e13a2b6ae45303029d06e347/protocol/messenger_sync_raw_messages.go#L22) for receiver, but what about sender? how can it get installation data from receiver? the solution was endpoints `receiveInstallation/sendInstallation` was created to resolve this. So it does  implemented exchanging installation data 
in this PR, otherwise [relate test 1](https://github.com/status-im/status-go/blob/6cbeedd880fd76a9e13a2b6ae45303029d06e347/server/pairing/sync_device_test.go#L280-L281) and [relate test 2](https://github.com/status-im/status-go/blob/6cbeedd880fd76a9e13a2b6ae45303029d06e347/server/pairing/sync_device_test.go#L194-L195) would fail.

Reply:
Ah, No I didn't have any objection with specific server endpoints to send / receive the installation data. I didn't realise that the installation data of the sender was sent in a previous call. I thought that this PR introduced all installation data exchange, rather than equalising the exchange of installation data.

Your tests make this point very clear. Thank you for your patience 😆.